tests/fixtures/preprocess-addon/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const Filter = require('broccoli-persistent-filter');
+const funnel = require('broccoli-funnel');
+const path = require('path');
+
+class Awk extends Filter {
+  constructor(inputNode, searchReplaceObj) {
+    super(inputNode, {});
+    this.searchReplaceObj = searchReplaceObj;
+  }
+
+  processString(content) {
+    let modifiedContent = content;
+
+    Object.entries(this.searchReplaceObj).forEach(([search, replace]) => {
+      modifiedContent = modifiedContent.replace(search, replace);
+    });
+
+    return modifiedContent;
+  }
+}
+
+module.exports = {
+  name: require('./package').name,
+
+  setupPreprocessorRegistry(type, registry) {
+    registry.add('css', {
+      name: this.name,
+      ext: 'css',
+      toTree: (tree, inputPath, outputPath) => {
+        console.log('inputPath: ' + inputPath);
```

Comment:
I assume you don't actually want to commit these console.logs in this file?

Reply:
No I am trying to debug the windows path issue